commit 9ad0cc7792fc1f0e0fc8985ba8810362707b9b20 (HEAD -> master)
Author: John Levon <john.levon@joyent.com>
Date:   2019-10-07T12:25:35+00:00 (11 hours ago)
    
    OS-7945 kvm needs to be built with retpolines
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Michael Zeller <mike.zeller@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/Makefile b/Makefile
index cee6818..71245cc 100644
--- a/Makefile
+++ b/Makefile
@@ -123,7 +123,9 @@ KERNEL_CFLAGS = \
 #
 ifneq ($(PRIMARY_COMPILER_VER),4)
 KERNEL_CFLAGS += \
-	-fno-shrink-wrap
+	-fno-shrink-wrap \
+	-mindirect-branch=thunk-extern \
+	-mindirect-branch-register
 endif
 
 USER_CFLAGS = \
